src/data/markdown/docs/02 javascript api/13 k6-redis.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "k6/experimental/redis"
+excerpt: "k6 Redis experimental API"
+---
+
+<Blockquote mod='info'>
+
+#### This module is experimental, use at your own risk
+
+**This module is experimental**. It is not guaranteed to be stable, and may change without notice. **Use at your own risk**.
+While will we intend to keep this module as simple and stable as possible, we may need to add more features or introduce breaking changes in the future, and won't refrain from doing so until we release this module as a "stable" one. Please kindly provide us your user feedback, and open an issue or pull request if you have any suggestions.
+</Blockquote>
+
+The [Redis](https://redis.io/) module for [k6](https://github.com/grafana/k6) provides a client library allowing users to interact with Redis, directly from their k6 scripts. It was built with the objective to enable users to either load test their Redis infrastructure, or use Redis as a data store backing their load tests' script logic.
+
+The module's API intends to be quite thorough and extensive, but is still limited to a set of functionalities the k6 team judged useful in the context of k6. The intent of this module isn't to provide a full-fledged Redis client, integrating the whole Redis API, but rather to empower our users to use Redis in use-cases we believe are relevant to k6. 
+
+This module is **not stable**, and remains open to changes. Thus, if any Redis functionalities were to be missing for your use-case, feel free to open an issue or pull request.
+
+
+### Notes on usage
+
+The `Client` exposes a [promise](https://javascript.info/promise-basics)-based API. Unlike most other current k6 modules and extensions, who operate in a synchronous manner,
+the Redis `Client` operates in an asynchronous manner. In practice, this means that using the Redis `Client`'s methods won't block the execution of the test, and that the test will continue to run even if the Redis `Client` is not ready to respond to the request.
+
+One potential caveat introduced by this new execution model, is that to depend on operations against Redis, all following operations should be made in the context of a [promise chain](https://javascript.info/promise-chaining). As demonstrated in the examples below, whenever there is a dependency on a Redis operation result or return value, the operation, should be wrapped in a promise itself. That way, a user can perform asynchronous interactions with Redis in a seemingly synchronous manner.
+
+## API
+
+| Class                                       | Description                                                                                    |
+| :------------------------------------------ | :--------------------------------------------------------------------------------------------- |
+| [Client](/javascript-api/k6-redis/client)   | Client exposing allowing interactions with Redis.                                              |
```

Comment:
> Client exposing allowing interactions with Redis.

Not sure here. Maybe:

Client that exposes allowed interactions with Redis.

Reply:
You're right, direct form is probably better here indeed!